d2js/js/src/index.d.ts

```diff
@@ -0,0 +1,148 @@
+declare module "@terrastruct/d2" {
+  interface Options {
+    /**
+     * @default 0
+     * Set the diagram theme ID.
+     */
+    theme?: number;
```

Comment:
Isn't it `themeID` https://github.com/terrastruct/d2/pull/2343/files#diff-08c4751b9ca6fc0c4b171cd225d3ac4b95b96874ffad8ae818ce22134b7bc524R7 ?